src/applications/caregivers/components/PreSubmitInfo/index.jsx

```diff
@@ -1,61 +1,16 @@
 import React, { useEffect, useState } from 'react';
-import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
-import SignatureInput from 'applications/caregivers/components/PreSubmitInfo/components/SignatureInput';
+import SignatureCheckbox from './components/SignatureBox';
 
-// single checkbox
-const SignatureCheckbox = ({
-  fullName,
-  label,
-  children,
-  signSignature,
-  signatures,
-}) => {
-  const [isSigned, setIsSigned] = useState(false);
-  const [isChecked, setIsChecked] = useState(false);
-  const isSignatureComplete = isSigned && isChecked;
+const PreSubmitCheckboxGroup = ({ onSectionComplete, formData }) => {
+  const veteranLabel = "Enter Veteran's or service member's full name";
+  const primaryLabel = "Enter Primary Family Caregiver's full name";
+  const secondaryOneLabel = "Enter Secondary Family Caregiver's full name";
+  const secondaryTwoLabel = "Enter Secondary Family Caregiver's (2) full name";
```

Comment:
Not a huge deal, but these apostrophes need to be the typographical version (`’` or `&apos;` in JSX)